Prog/AddNew/TOPA_data_to_DC_data.sas

```diff
@@ -132,23 +162,80 @@ proc sql noprint;
   select unique 
     coalesce( TOPA_SSL.SSL, RealProp.SSL ) as SSL, /** Matching variables **/
 	/** obs where sale date is after the later of CASD data and offer of sale data **/
-	saledate - offer_sale_date as days_notice_to_sale,
+	saledate - u_offer_sale_date as u_days_notice_to_sale label="Number of days from offer of sale to actual sale (Urban created var)",
 	TOPA_SSL.ID, /** Other vars you want to keep from the two data sets, separated by commas **/
-	TOPA_SSL.CASD_date, TOPA_SSL.offer_sale_date,
+	TOPA_SSL.u_CASD_date, TOPA_SSL.u_offer_sale_date,
 	TOPA_SSL.Anc2012, TOPA_SSL.cluster2017, TOPA_SSL.Geo2020, 
 	TOPA_SSL.GeoBg2020, TOPA_SSL.GeoBlk2020, TOPA_SSL.Psa2012,
 	TOPA_SSL.VoterPre2012, TOPA_SSL.Ward2022, TOPA_SSL.Ward2012,
     RealProp.SSL, RealProp.SALEPRICE, RealProp.saleprice_prev, RealProp.SALEDATE, RealProp.saledate_prev, RealProp.Ownername_full, RealProp.ownername_full_prev, RealProp.ui_proptype,
-	RealProp.address1, RealProp.address2, RealProp.address3, RealProp.address1_prev, RealProp.address2_prev, RealProp.address3_prev  
+	RealProp.address1, RealProp.address2, RealProp.address3, RealProp.address1_prev, RealProp.address2_prev, RealProp.address3_prev, RealProp.premiseadd, RealProp.hstd_code  
     from TOPA_SSL (where=(not(missing(SSL)))) as TOPA_SSL
       left join RealProp.Sales_master as realprop    /** Left join = only keep obs that are in TOPA_geocoded **/
   on TOPA_SSL.SSL = realprop.SSL   /** This is the condition you are matching on **/
-  where SALEDATE > max(CASD_date, offer_sale_date) /** obs where sale date is after the later of CASD data and offer of sale data **/
+  where SALEDATE > max(u_CASD_date, u_offer_sale_date) /** obs where sale date is after the later of CASD data and offer of sale data **/
```

Comment:
@eburton9 Sorry, I forgot to mention this in my earlier message, but let's just take out this where clause for the real property data. We want to see prior sales, too, and we can figure out the sequencing using the method that I shared in the email I sent on Friday. Thanks!